### PR TITLE
Update window post

### PR DESCRIPTION
**Pull request by turuslan**
### Description of the Change
Update window post scheduler

### CODE REVIEW

File: core/vm/actor/builtin/v0/miner/policy.hpp [hunk 1]
```diff
@@ -19,7 +19,7 @@ namespace fc::vm::actor::builtin::v0::miner {
   using primitives::TokenAmount;
   using primitives::sector::RegisteredProof;
 
-  constexpr size_t kEpochDurationSeconds{25};
+  constexpr size_t kEpochDurationSeconds{30};
```
Comment (Alexey-N-Chernyshov) [inline core/vm/actor/builtin/v0/miner/policy.hpp:22]: kEpochDurationSeconds  is already declared in "const.hpp".

File: test/core/primitives/rle_bitset/rle_bitset_test.cpp [hunk 2]
```diff
@@ -17,3 +17,17 @@ TEST(RleBitsetTest, RleBitsetCbor) {
   using fc::primitives::RleBitset;
   expectEncodeAndReencode(RleBitset{2, 7}, "43504a01"_unhex);
 }
+
+TEST(RleBitsetTest, Runs) {
```
Comment (Alexey-N-Chernyshov) [inline test/core/primitives/rle_bitset/rle_bitset_test.cpp:21]: please, add test comment

File: core/vm/actor/builtin/v0/miner/policy.hpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply core/vm/actor/builtin/v0/miner/policy.hpp:22]: Please, take a look at "core/clock/chain_epoch_clock.hpp:27" at `kEpochDuration{15}`. Is it correct value?

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: core/vm/actor/builtin/v0/miner/policy.hpp (hunk 1 above)
Comment (turuslan) [reply core/vm/actor/builtin/v0/miner/policy.hpp:22]: Miner's `kEpochDurationSeconds` is constexpr, but "const.hpp" could be mutable (using config), so i'll leave it for now

File: core/api/api.hpp [hunk 3]
```diff
@@ -281,12 +276,26 @@ namespace fc::api {
...
+  };
+
   struct SectorLocation {
     uint64_t deadline;
     uint64_t partition;
   };
 
-  constexpr None kPushNoSpec{};
+  struct MessageSendSpec {
+    static TokenAmount maxFee(const boost::optional<MessageSendSpec> &spec) {
```
Comment (artyom-yurin) [inline core/api/api.hpp:289]: Why it is in the class?

**Review by artyom-yurin — APPROVED** [review]

**Pull request merged**